File: FloorDeJong1/beginners/src/main/java/Circle.java
public class Circle implements Figure2D {
    private double radius;

    public Circle(double radius) {
        this.radius = radius;
    }

    @Override
    public double getArea() {
        return Math.PI * Math.pow(this.radius, 2);
    }

    @Override
    public double getCircumference() {
        return 2*Math.PI*this.radius;
    }

    public double getRadius() {
        return radius;
    }

    public void setRadius(int radius) {
        this.radius = radius;
    }
}

File: frank/GEP Coding Challenge 2: OOP/beginner/challenge #1/domain/Circle.py
from math import pow, pi
from domain.IArea import IArea


class Circle(IArea):

    def calc_area(self, value):
        return pi * pow(value/2, 2)


# c = Circle()
# for x in range( 10 ):
#    print( x, c.calc_area( x ) )
